cranelift-codegen/src/isa/mod.rs

```diff
@@ -371,4 +371,10 @@ pub trait TargetIsa: fmt::Display + Sync {
 
     /// Emit a whole function into memory.
     fn emit_function_to_memory(&self, func: &ir::Function, sink: &mut binemit::MemoryCodeSink);
+
+    /// Get the value for the condition CF=1.
```

Comment:
This comment is in target-independent code, so we should say something like "the condition for unsigned overflow" instead of referencing "CF". In fact, we should probably say `unsigned` somewhere in the function name too, like `unsigned_add_overflow_condition`, just to be unambiguous.